internal/engine/podlogmanager.go

```diff
@@ -146,38 +145,43 @@ func (m *PodLogManager) consumeLogs(watch PodLogWatch, st store.RStore) {
 		watch.cancel()
```

Comment:
i'm pretty sure this cancel() and terminationTime is obsolete now, since the only way to exit the loop is to cancel the watch and remove it from the watches array